app/src/pages/Login.tsx

```diff
@@ -1,64 +1,147 @@
-import React from "react";
-
 import styled from "styled-components";
 
-const BrandText = styled.div`
-  font-family: "Open Sans";
-  font-style: normal;
+import Logo from "../assets/logo.svg";
+import { BrandText } from "../styled-components";
+
+const PageContainer = styled.div`
+  background: #ffffff;
+  width: 367px;
+  height: 611px;
+  padding: 14px;
+`;
+
+const HeaderText = styled(BrandText)<{ fontSize?: number }>`
+  letter-spacing: 0.15em;
   font-weight: 700;
-  font-size: 14px;
-  line-height: 19px;
-  text-transform: uppercase;
+  padding-bottom: 4px;
 
-  color: #a8192e;
+  ${(props) => props.fontSize && `font-size: ${props.fontSize}px`};
 `;
 
-const BrandSubtext = styled.div`
-  font-family: "Open Sans";
+const SubheaderText = styled(BrandText)`
   font-style: italic;
-  font-weight: 300;
-  font-size: 10px;
-  line-height: 14px;
   letter-spacing: 0.15em;
-  text-transform: uppercase;
+  font-size: 10px;
+  position: relative;
+  max-width: 600px;
+  text-align: center;
+  margin: auto;
+
+  &:before {
+    content: "";
+    width: 54px;
+    height: 0.5px;
+    background: #a8192e;
+    left: 16px;
+    top: 50%;
+    position: absolute;
+  }
 
-  color: #a8192e;
+  &:after {
+    content: "";
+    width: 54px;
+    height: 0.5px;
+    background: #a8192e;
+    right: 16px;
+    top: 50%;
+    position: absolute;
+  }
 `;
 
-const CtaText = styled.div`
-  font-family: "Open Sans";
-  font-style: normal;
-  font-size: 14px;
-  line-height: 19px;
-  text-align: center;
+const CtaText = styled(BrandText)`
+  color: #000000;
+  padding-top: 24px;
+`;
 
+const InputWrapper = styled.div`
+  padding-top: 32px;
+  width: 331px;
+  margin: auto;
+`;
+
+const InputLabel = styled(BrandText)`
   color: #000000;
+  text-align: left;
+`;
+
+const EmailInput = styled.input`
+  width: 331px;
+  height: 50px;
+  box-sizing: border-box;
+  background: #ffffff;
+  border: 1px solid #dadada;
+  border-radius: 5px;
+  margin-top: 12px;
+`;
+
+const ButtonWrapper = styled.button`
+  background: #8b8b8b;
+  border-radius: 50px;
+  width: 267px;
+  height: 45px;
+  margin-top: 74px;
+  margin-bottom: 80px;
 `;
 
-const EmailInput = styled.div`
-  display: flex;
+const ButtonText = styled(BrandText)`
+  letter-spacing: 0.15em;
+  color: #ffffff;
```

Comment:
cant find the specific reference but the text color is also white in the email box? is this expected

Reply:
@stanzheng good catch. I’m gonna open another PR for the red card styling and will include the fix for the text